decidim-bulletin_board-app/cypress/integration/sandbox/election.page.js

```diff
@@ -141,13 +141,31 @@ export class ElectionPage {
   }
 
   /**
-   * Audit a vote.
+   * Encrypt a vote.
    *
    * @returns {undefined}
    */
-  auditVote() {
+  encryptVote() {
     cy.findByText("Vote").click();
     cy.findByText("Encrypt vote").should("be.visible").click();
+  }
+
+  /**
+   * Assert that the ballot hash is present
+   *
+   * @returns {undefined}
+   */
+  assertBallotHashIsPresent() {
+    cy.findByText("Encrypt vote").should("be.disabled");
+    cy.get(".ballot-hash").should("include.text", "Your ballot identifier is:");
```

Comment:
Avoid using selectors if possible. I think it's better to just use `cy.findbyText("Your ballot identifier is:").should("be.visible")`.